src/test_trie.py

```diff
@@ -0,0 +1,196 @@
+"""Tests for Python implementation of Trie."""
+import pytest
+from trie import Trie, Node
+
+
+@pytest.fixture
+def init_empty_trie():
+    """Init empty trie."""
+    return Trie()
+
+
+@pytest.fixture
+def init_trie_one_word():
+    """Init empty trie."""
+    new_trie = Trie()
+    new_trie.insert('cake')
+    return new_trie
+
+
+@pytest.fixture
+def init_trie_three_words_shared_letters():
+    """Init empty trie."""
+    new_trie = Trie()
+    new_trie.insert('be')
+    new_trie.insert('bee')
+    new_trie.insert('bell')
+    return new_trie
+
+
+@pytest.fixture
+def init_jumbo_list():
+    """Fixture for 250k word test."""
+    # jumbo_trie = Trie()
+    word_file = open('/usr/share/dict/words', 'r')
```

Comment:
There is a cleaner way to do these few lines, take a look into the `with open(<filename>, 'r'):` command. Using this you don't have to explicitly close the file.